app/helpers/app_text_helper.rb

```diff
@@ -0,0 +1,16 @@
+module AppTextHelper
+  def action_text(key, options = {})
+    named_text(:actions, key, options)
+  end
+
+  def thead_text(key, options = {})
+    named_text(:table_headings, key, options)
+  end
+
+  private
+
+  def named_text(text_type, key, options = {})
+    namespace = controller.text_namespace
```

Comment:
Will this only get used with the manage users controller? Wondering if it is worth adding a superclass method that throws a `NotImplemented` error or something, just incase this is mixed into a controller that doesn't define `text_namespace` 🤔 

If we did this are there other tables we could do this in?